CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ option(ENABLE_ADE_TESTING      "Build tests, require google test" OFF)
 option(BUILD_ADE_TUTORIAL      "Build tutorial samples"           OFF)
 option(FORCE_ADE_ASSERTS       "Always enable ADE_ASSERT"         OFF)
 option(BUILD_ADE_DOCUMENTATION "Build doxygen documentation"      OFF)
+option(BUILD_STATIC "Build static libs" OFF)
```

Comment:
> BUILD_STATIC

Why don't reuse CMake's `BUILD_SHARED_LIBS` variable?

Reply:
+1

Reply:
Actually, I add this option because I want ade to link static CRT so as to avoid linking vcruntime140.dll and msvcp140.dll which are not universal dll according to Microsoft API scanner.